esmvaltool/diag_scripts/rainfarm/rainfarm_parameters.r

```diff
@@ -0,0 +1,21 @@
+#####################################################################
+#
+#  RainFARM advanced configuration file
+#
+#  About: configuration file with advanced settings for ESMValTool RainFARM namelist.
+#
+#####################################################################
+
+force_processing = T # TRUE to rewrite existing output file
+
+# RainFARM options 
+# Note - RainFARM usage:  rfarm [-s SLOPE] [-e NENS] [-n NF] [-r REGION] [-w WEIGHTS]
+#                               [-o OUTFILE] [-v VARNAME] [-g] [-c]
+# set FALSE to void argument
+weights_climo =     # climatology file to be used to calculate weights (calling rfweights) 
+          "/work/datasets/obs/unsorted/WORLDCLIM/prec/prec_06_16_central_europe.nc" 
```

Comment:
Shouldn't this settings be part of the recipe now with version2 instead of a dedicated configuration file like it used to be in version1.

Reply:
See response above.